mutt/queue.h

```diff
@@ -354,6 +354,12 @@ struct {								\
 	   (var);							\
 	   (var) = STAILQ_NEXT((var), field))
 
+#define	STAILQ_FOREACH_ENUMERATE(var, index, head, field) \
```

Comment:
Hmm... now this is tricky.  `mutt/queue.h` isn't ours to alter.
It's a standard used by Linux and the BSDs.
It's a nice invention, but for now I'd prefer to use `STAILQ_FOREACH()` and a simple `index++`

Reply:
Ahh, I was not aware of that! Will change.